src/apsbits/utils/make_devices.py

```diff
@@ -21,16 +21,27 @@
 from apstools.utils import dynamic_import
 from bluesky import plan_stubs as bps
 
-from apsbits.utils.aps_functions import host_on_aps_subnet
-from apsbits.utils.config_loaders import get_config
-from apsbits.utils.config_loaders import load_config_yaml
-from apsbits.utils.controls_setup import oregistry  # noqa: F401
+from ..utils.aps_functions import host_on_aps_subnet
+from ..utils.config_loaders import get_config
+from ..utils.config_loaders import load_config_yaml
+from ..utils.controls_setup import oregistry  # noqa: F401
 
 logger = logging.getLogger(__name__)
 logger.bsdev(__file__)
 
+MAIN_NAMESPACE = "__main__"
```

Comment:
Is this ever NOT going to be main? I moved out of the top of the file because it implies this is a system level variable.

Reply:
I don't think we should ever import `__main__` anywhere else unless we are
creating different namespaces. Either way, not very problematic here.